File: app/src/main/java/koushur/kashmirievents/database/entity/MonthDataEntity.java
package koushur.kashmirievents.database.entity;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "monthData")
public class MonthDataEntity {
    @NonNull
    @PrimaryKey
    public String date = "abc";
    @NonNull
    public String events = "abc";
}

File: app/src/main/java/koushur/kashmirievents/database/entity/MonthDataEntity.kt
package koushur.kashmirievents.database.entity

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.google.gson.annotations.SerializedName

@Entity(tableName = "monthData")
data class MonthDataEntity(
    @PrimaryKey
    @SerializedName("date")
    val date: String,

    @SerializedName("events")
    val events: String = "",

    @SerializedName("imp")
    val imp: Int = 0
)